lib/predictions/repo.ex

```diff
@@ -24,14 +24,15 @@ defmodule Predictions.Repo do
     opts
     |> add_all_optional_params()
     |> cache_fetch()
-    |> filter_by_min_time(Keyword.get(opts, :min_time))
+    |> filter_predictions(Keyword.get(opts, :min_time))
     |> load_from_other_repos
   end
 
   def all_no_cache(opts) when is_list(opts) and opts != [] do
     opts
     |> add_all_optional_params()
     |> fetch()
+    |> filter_predictions(nil)
```

Comment:
Using a default argument of nil rather than passing in nil makes both the function definition and it's application much clearer and cleaner.